jobs/integration/validation.py

```diff
@@ -2254,15 +2305,27 @@ async def nagios(model, tools):
         app = await model.deploy("nagios", series="bionic")
         await app.expose()
     if "nrpe" not in model.applications:
+        excludes = [
+            "/snap/",
+            "/sys/fs/cgroup",
+            "/run/containerd",
+            "/var/lib/docker",
+            "/run/credentials",
+            "/run/systemd/incoming",
+        ]
```

Comment:
this overrides the defaut `nrpe` config `ro_filesystem_excludes`. 

it comes with `/snap/,/sys/fs/cgroup,/run/containerd,/var/lib/docker` by default
in jammy, the test wouldn't setup as stable until we ignored the other read-only filesystems that nagios could see:
`/run/credentials` and `/run/systemd/incoming`.  

TBH -- i found `/run/credentials` mounted in read-only on a jammy file system.  I never found `/run/systemd/incoming` in `/proc/mounts` so i have no clue why nagios is seeing it.  But in eithercase, i'm ok if that filesystem is readonly during this tests -- so lets exclude it!